features/git-town-append/hack-push-flag.feature

```diff
@@ -0,0 +1,49 @@
+Feature: push branch to remote upon creation
```

Comment:
User story:

> When creating a new feature branch and having enough CI server bandwidth for an extra CI run
I want it to be pushed to the CI server right away
So that I can push and pull from the remote branch right away without having to run "git sync" first